src/electron/device-disconnected-popup/device-disconnected-popup.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/fonts.scss';
+
+.device-disconnected-popup {
+    .title-container {
+        display: flex;
+
+        svg {
+            align-self: center;
+        }
+    }
+
+    :global {
```

Comment:
I've had some hit or miss usage with:

```
:global {
    ... // css here
}
```

Using:

```
:global(.class-name) { 
    .... // css here
}
```
seems to give more consistent behavior. I'm not fully sure why ...

Reply:
@waabid, this also seems to add the :global attribute to css file. will this affect the style of the headers outside device-disconnected-popup as well? If so, then we should avoid using this.

Reply:
Thanks! Will update

Reply:
> @waabid, this also seems to add the :global attribute to css file. will this affect the style of the headers outside device-disconnected-popup as well? If so, then we should avoid using this.

I'm not seeing this behavior?

Reply:
I pulled back from using css modules on this PR as we saw some weird behavior and css module is not the main purpose of the PR